front-end/studio/src/app/pages/apis/create/_components/create-form.component.ts

```diff
@@ -68,8 +68,8 @@ export class CreateApiFormComponent {
 
     public typeOptions(): DropDownOption[] {
         return [
-            { name: "Open API 2.0 (Swagger)", value: "2.0"},
-            { name: "Open API 3.0.2", value: "3.0.2"}
+            new Value("Open API 2.0 (Swagger)", "2.0"),
+            new Value("Open API 3.0.2", "3.0.2")
```

Comment:
I was unsure about this change (adding methods to the class) but this is pretty nice. :)